src/iosrv/srv.c

```diff
@@ -321,6 +323,17 @@ dss_nvme_poll_ult(void *args)
 
 	D_ASSERT(dx->dx_main_xs);
 	while (!dss_xstream_exiting(dx)) {
+		uint64_t now = 0;
```

Comment:
minor, need not = 0. PROGRESS_MESG_INTV maybe PROGRESS_WARN_INTV and D_WARN

not very sure if it is really helpful and slight overhead added (daos_gettime_coarse), or maybe change to check the time for every 10K progress?

Reply:
Oh, I do not want to generate the warning, just a message I can use inside the daos_log to see when the last progress is called, so to know if there is some "deadlock" ult hold the CPU cycle.

get time_coarse  use CLOCK_MONOTONIC, so the performance should not be an issue?